tests/e2e/restart_test.go

```diff
@@ -330,7 +330,8 @@ func (s *RestartTestData) TestDuplicateVolumeName(c *C) {
 	c.Assert(err, IsNil, Commentf(out))
 
 	// Cleanup volume on the second datastore
-	out, err = dockercli.DeleteVolume(s.config.DockerHosts[1], s.volumeName+"@"+ds2)
+	volName := "'" + s.volumeName + "@" + ds2 + "'"
```

Comment:
Suggest making a util function CreateFQVN() that handles adding the _'_ to create a full volume name with its datastore. So this code isn't repeated. The function returns a string that can be used in any command string.